atom/browser/ui/views/menu_bar.cc

```diff
@@ -72,6 +72,7 @@ void MenuBar::SetMenu(AtomMenuModel* model) {
     button->SetTextColor(views::Button::STATE_PRESSED, highlight_color_);
     button->SetTextColor(views::Button::STATE_HOVERED, hover_color_);
     button->SetUnderlineColor(enabled_color_);
+    button->SetEnabledTextColors(SK_ColorBLACK);
```

Comment:
Let me know what you prefer to do here, I think we could just remove getting the hovered and highlight color which returns white on white and doesn't work properly.